tools/coqdep/lib/loadpath.ml

```diff
@@ -157,11 +154,19 @@ let rec cuts recur = function
     ([],if recur then suffixes true l else [true,l]) ::
     List.map (fun (fromtail,suffixes) -> (dir::fromtail,suffixes)) (cuts true tail)
 
+let warning_ml_clash' =
```

Comment:
BTW usually I think we would shadow the name instead of addin `'`

Reply:
Agreed, that was not my code initially.